### ISSUE TITLE
Wappalyzer returns No response from server since version 6.3.0

### ISSUE THREAD

**Issue by sejoker** [AliasIO/wappalyzer#3424]
**Platform**
Which platform does the bug apply to?
MacOS 10.15.6, node v12.14.0,  NPM 6.13.4

**Describe the bug**
Since commit https://github.com/AliasIO/wappalyzer/commit/5de3c5299e8f85c83ddb667d357b79d1a9d424c5 
```
wappalyzer https://yadakpars.com
```
returns:
```
{"urls":{"https://yadakpars.com/":{"status":0,"error":"No response from server"}},"technologies":[]}
```

**To Reproduce**
1. npm i wappalyzer -g
2. `wappalyzer https://yadakpars.com`

**Expected behavior**
Able to fetch technologies list when calling:
```
wappalyzer https://yadakpars.com
```

**Additional context**
The problematic puppeteer settings which break everything is:
```
let chromiumArgs = [
 '--user-data-dir=/tmp/chromium'
```

**Comment by AliasIO** [comment]
What is the output of `wappalyzer https://yadakpars.com -d`?

**Comment by sejoker** [comment]
```
wappalyzer | log | driver | Launching browser...
wappalyzer | log | page | Navigate to https://yadakpars.com/
wappalyzer | log | driver | Page closed
wappalyzer | error | driver | Error: No response from server
    at Site.goto (/Users/user/.nvm/versions/node/v12.14.0/lib/node_modules/wappalyzer/driver.js:753:13)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async Site.analyze (/Users/user/.nvm/versions/node/v12.14.0/lib/node_modules/wappalyzer/driver.js:767:21)
    at async /Users/user/.nvm/versions/node/v12.14.0/lib/node_modules/wappalyzer/cli.js:87:21
{"urls":{"https://yadakpars.com/":{"status":0,"error":"No response from server"}},"technologies":[]}
wappalyzer | log | driver | Browser disconnected
wappalyzer | log | driver | Browser closed
```

**Comment by sejoker** [comment]
Same stack for node 14.13.0.

**Comment by AliasIO** [comment]
Are you using the latest version of Wappalyzer? When I run the command I do get results.

**Comment by kyletaylored** [comment]
Seems to be resolved on 6.5.22.
